### ISSUE TITLE
Research Antlet Cloning Process

### ISSUE THREAD

**Issue by jwajer** [jwajer/capstone#4]
If there's any time remaining, I want to see if I can learn how to clone Antlets.  Cloning Antlets would make it far easier for a study abroad site administrator to create Antlets for an upcoming class.  Only one Antlet actually needs to be created and can just be cloned for other students.

**Comment by jwajer** [comment]
Cloning the Antlets appear to be as easy as clicking Clone this Antlet in the antMan console.  I'll test that out now to see if it is indeed that easy.

**Comment by jwajer** [comment]
Cloning for Linux was indeed as easy as selecting Clone this Antlet.  I'll check Windows now to make sure it's that easy as well.  Before cloning, I installed the net-tools and tree packages on the original CentOS Antlet.  After cloning, I checked the cloned Antlet and both packages were installed.

**Comment by jwajer** [comment]
Cloning Windows Antlets was just as simple as it was for Linux.  Scalability for the Antsle appliance that I have could potentially be a concern due to the 4 Core CPU, and it's impossible to give an Antlet less than 1 virtual CPU core.  With three of the four Antlets with their consoles open, I'm seeing CPU usage hover around 30%, with spikes as high as 50% on occasion.  Prior to cloning the Windows Antlet, I downloaded Wireshark onto the machine.  After cloning the Antlet, I checked the new Windows Antlet and Wireshark was downloaded on it, with the same download time listed for both the original Antlet and the cloned Antlet.
